humble/distribution.yaml

```diff
@@ -652,6 +652,12 @@ repositories:
       url: https://github.com/autowarefoundation/autoware_cmake.git
       version: rolling
     status: developed
+  autoware_msgs:
+    source:
+      type: git
+      url: https://github.com/autowarefoundation/autoware_msgs.git
```

Comment:
In [Noetic](https://github.com/ros/rosdistro/blob/72eeaec8e84f147feed051717ed324edc006a60c/noetic/distribution.yaml#L389), this key comes from a different repository (https://github.com/autoware-ai/messages.git).  What's the relationship between these repositories?

Reply:
It was old version of the `autoware_msgs`. The ROS 1 Autoware.AI is no longer supported. That's why I removed it.
https://github.com/autowarefoundation/autoware?tab=readme-ov-file#using-autowareai